layers/layer_options.cpp

```diff
@@ -578,6 +579,15 @@ void ProcessConfigAndEnvSettings(ConfigAndEnvSettings *settings_data) {
         SetValidationFlags(settings_data->disables, validation_flags_ext);
     }
 
+    // if app is setting VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_EXT, we can use this to disable it for debugging
+    if (vkuHasLayerSetting(layer_setting_set, VK_LAYER_GPUAV_DEBUG_DISABLE_ALL)) {
+        bool disable_gpu = false;
+        vkuGetLayerSettingValue(layer_setting_set, VK_LAYER_GPUAV_DEBUG_DISABLE_ALL, disable_gpu);
+        if (disable_gpu) {
+            settings_data->enables[gpu_validation] = false;
```

Comment:
While in here （￣︶￣）↗　
```suggestion
            settings_data->enables[gpuav] = false;
```

Reply:
this is a out of scope change

![image](https://github.com/KhronosGroup/Vulkan-ValidationLayers/assets/115671160/506fc80e-2049-4dca-932e-e20c50286486)

Reply:
![4pk9q1-4122960734](https://github.com/KhronosGroup/Vulkan-ValidationLayers/assets/111733687/26696d5a-a938-4342-ab43-f71e57b8cdf3)